frontends/election-manager/package.json

```diff
@@ -10,12 +10,12 @@
   "scripts": {
     "type-check": "tsc --build",
     "build": "pnpm type-check && vite build",
-    "build:watch": "tsc --build --watch",
     "build:stubs": "script/build-stubs fs:src/stubs/fs.ts glob:src/stubs/glob.ts os:src/stubs/os.ts",
     "format": "prettier '**/*.+(css|graphql|json|less|md|mdx|sass|scss|yaml|yml)' --write",
     "lint": "pnpm type-check && eslint . && pnpm stylelint:run",
     "lint:fix": "pnpm type-check && eslint . --fix && pnpm stylelint:run:fix",
-    "start": "vite",
+    "start": "concurrently --names 'frontend,libs' --prefix-colors 'green,yellow' --kill-others  \"vite --clearScreen false\" \"tsc --build --watch --preserveWatchOutput\"",
+    "start:all": "concurrently --names 'frontend,libs,backend,smartcard' --prefix-colors 'green,yellow,magenta,cyan' --hide 'smartcard' --kill-others \"vite --clearScreen false\" \"tsc --build --watch --preserveWatchOutput\" \"pnpm -w run:smartcards\" \"pnpm dev --prefix ../../services/admin\"",
```

Comment:
Also `converter-ms-sems`?

Reply:
Since we have multiple converters (and all the concurrently commands are spun up and killed together), I thought to leave converters. Otherwise I could make this into a proper script which takes a converter name as an argument.